spec/models/like_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
```

Comment:
- [ ] Kindly test for the authorId where it is valid. I see you have created the user, post, and like. I would recommend testing whether the like belongs to the author who created it or not.  You have tested for its invalid part, now you can test for the valid part.